src/collect-settings.rst

```diff
@@ -1,82 +1,133 @@
+To access General Settings:
+  :menuselection:`⋮ --> General Settings`
 
 .. image:: /img/collect-settings/general-settings.*
   :alt: General settings
   :class: device-screen-vertical
 
-It allows us to configure the following:
-
 .. _server-settings:
 
 Server Settings
 ~~~~~~~~~~~~~~~~~
 
+Server settings :doc:`control the connection to <collect-connect>` 
+an :doc:`Aggregate <aggregate-intro>` or :doc:`openrosa` server
+or a :doc:`Google Drive account <collect-connect-google>`. 
+
+To access Server Settings:  
+  :menuselection:`⋮ --> General Settings --> Server` 
+
+  
 .. image:: /img/collect-settings/server-settings.*
   :alt: Server settings
   :class: device-screen-vertical
 
-:menuselection:`Server` settings manage the connection to an ODK Aggregate server or Google Drive account for form management.
-
-See :doc:`collect-connect` for more details.
+.. seealso:: :doc:`collect-connect`
 
 .. _interface-settings:
 
 User Interface Settings
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
+User Interface settings control Collect's appearance and behavior. 
+
+To access User Interface settings:
+  :menuselection:`⋮ --> General Settings --> User Interface` 
+
 .. image:: /img/collect-settings/ui-settings.*
   :alt: User Interface settings
   :class: device-screen-vertical
 
-:menuselection:`User Interface` settings allow us to customize the way our app looks and behaves. It includes settings for:
+:guilabel:`Language` 
+  Sets the display language.
 
-- :guilabel:`Language` - Allows us to select the device language from 49 available options.
-- :guilabel:`Text font size` - Lets us select the font-size.
-- :guilabel:`Navigation` (Swiping or Buttons) - Allows us to select the navigation method(swipes, buttons or both).
-- :guilabel:`Splash Screen` - An image to display when the Collect application loads.
+:guilabel:`Text font size`
+  Sets the display font size.
+    
+:guilabel:`Navigation` 
+  Sets form navigation style for moving between questions.
+  
+  Options:
+
+  - Horizontal swiping
+  - Forward and back buttons
+  - Both
 
-:menuselection:`User Interface` settings page also consists of the Mapping Settings. 
+:guilabel:`Splash Screen`
+  Sets an image to display while Collect loads.
 
 .. _mapping-settings:
 
 .. rubric:: Mapping
 
-Map settings are included on the :menuselection:`User Interface` settings page.
-
-- :guilabel:`Mapping SDK` sets the app used for :ref:`geopoint`, :ref:`geoshape`, and :ref:`geotrace` :doc:`form-widgets`. Options are Google Maps (default) and OpenStreetMap.
-- :guilabel:`Basemap` sets the specific map used when opening the mapping app. 
+:guilabel:`Mapping SDK` 
+  Sets the app that will be used for 
+  form question types that require map integration.
+  
+  Options:
+  
+  - Google Maps (default)
+  - OpenStreetMap
+ 
+  .. seealso:: :ref:`geopoint`, :ref:`geoshape`, :ref:`geotrace` 
+  
+  
+:guilabel:`Basemap` 
+  Sets the map to be displayed when the mapping app is opened. 
```

Comment:
"mapping app" doesn't make sense. Maybe "when a widget with a mapping component is used."

Reply:
sounds reasonable